canonical-web-and-design/vanilla-framework
# Update the docs and examples for chips

## Done

Remove non-interactive chips from docs and examples.
Make sure all chips are using `<button>` if they are meant to be interacted with, and `<span>` is only used when chip contains a dismiss button.

Fixes #4187 
## QA

- Open [demo](https://vanilla-framework-4188.demos.haus/docs/patterns/chip)
- Make sure documentation and examples show "button" chips by default and only chips with dismiss button use span
- Check ALL of the chips examples to make sure they are correct
